test/object/object.js

```diff
@@ -156,4 +156,48 @@ function test(binding) {
     assert.strictEqual(binding.object.instanceOf({}, Ctor), false);
     assert.strictEqual(binding.object.instanceOf(null, Ctor), false);
   }
+
+  {
+    if (NAPI_CPP_EXCEPTIONS) {
```

Comment:
Maybe you can see if `'sum' in binding.object`?

Reply:
Also, this `if` statement seems to be inside an unnecessary additional block?

Reply:
@KevinEady Thank you for the comments, addressed! :)